kuma/authkeys/decorators.py

```diff
@@ -9,11 +9,12 @@
 
 
 def accepts_auth_key(func):
-    """Enable a view to accept an auth key via HTTP Basic Auth.
+    """
+    Enable a view to accept an auth key via HTTP Basic Auth.
```

Comment:
Ough, yeah. In general I like having a visual block on top of functions that moves the payload of the function away from the name. Also, one line docstrings are in my experience almost always a sign that docs are missing so I err on the side of promoting writing prose in there that just write short half sentences.

Since PEP 257 is ambiguous about this ("The summary line may be on the same line as the opening quotes or on the next line." https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) I'm following Django's example of putting the summary line one the next line as it was my school of Python style. I'm eager to follow that but I can understand if you're reluctant.